Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '~> 2.5'
+ruby '>= 2.5', '< 2.7'
```

Comment:
Note that `~> 2.5` is syntactic sugar for `>= 2.5, < 2.6`, so this is an even smaller change than it appears.